cl/opinion_page/templates/includes/de_filter.html

```diff
@@ -108,4 +108,48 @@
       </div>
     </form>
   </div>
+
+{% if docket.date_terminated is None %}
+  <p style="margin-top: 10px;">
+    <i class="fa fa-info-circle"></i> Docket entries on this page are sourced from PACER and may not be up to date. Learn more about our coverage <a href="{% url 'coverage_recap' %}">here</a>.
+  </p>
+  <p style="margin-bottom: 0;">
+    Notice an issue with the data displayed? Please <a href="{% url 'contact' %}">let us know</a>, and to further help improve public access to court records, consider <a href="https://free.law/recap/" class="alert-link">installing RECAP</a>—it's quick, easy, and free!
+  </p>
+{% endif %}
```

Comment:
Now that I'm seeing this, I think we should just show it on all pages, and maybe simplify it a bit to:

```suggestion
<p style="margin-top: 10px; margin-bottom: 0;">
  <i class="fa fa-info-circle"></i> Items on this page are sourced from <a href="https://www.pacer.gov">PACER</a> and <a href="https://free.law/recap/">RECAP</a> and may not be up to date. <a href="{% url 'coverage_recap' %}">Learn more about our coverage</a>.
</p>
```

That gets it on one line (I think), that we can just put on all the pages.